opsworks_custom_cookbooks/recipes/iptables.rb

```diff
@@ -0,0 +1,40 @@
+subnet_cidr = node[:deploy]['aviary'][:environment_variables][:SUBNET_CIDR]
+
+bash "configure_iptables" do
+  code <<-EOH
+    iptables -P INPUT ACCEPT
+    iptables -P FORWARD ACCEPT
+    iptables -P OUTPUT ACCEPT
+    iptables -F
+
+    iptables -P INPUT DROP
+    iptables -P FORWARD DROP
+    iptables -P OUTPUT DROP
+
+    iptables -A INPUT -i eth0 -p tcp --dport 22 -m state --state NEW,ESTABLISHED -j ACCEPT
+    iptables -A OUTPUT -o eth0 -p tcp --sport 22 -m state --state ESTABLISHED -j ACCEPT
+
+    iptables -A INPUT -i lo -j ACCEPT
+    iptables -A OUTPUT -o lo -j ACCEPT
+    iptables -A INPUT -i eth0 -p tcp --dport 80 -m state --state NEW,ESTABLISHED -j ACCEPT
```

Comment:
My understanding is that we don't want to accept port `80` or `443` for input on Aviary boxes.  It needs the outbound so that it can get out to the internet, but we aren't running servers there so we don't want inbound.

Does that sound correct?